js-exercises/minima/minima.js

```diff
@@ -0,0 +1,39 @@
+const isNumber = (num) => typeof num === 'number';
+
+function minima(numberOfElementsToReturn, array) {
+  const isValidNumber = isNumber(numberOfElementsToReturn);
+
+  if (!isValidNumber) {
+    throw new TypeError(`expected number , got ${typeof numberOfElementsToReturn}`);
+  }
+
+  if (numberOfElementsToReturn < 0) {
+    throw new Error('length must be greater or equal to zero.');
+  }
+
+  if (!Array.isArray(array)) {
+    throw new TypeError(`expected array , got ${typeof array}`);
+  }
+
+  const inputArray = array.filter(element => isNumber(element));
+  const sortedArray = inputArray.sort((firstNum, secondNum) => firstNum - secondNum);
```

Comment:
No need to declare one more variable to assign sorted array value. check the suggestion
```suggestion
  inputArray.sort((firstNum, secondNum) => firstNum - secondNum);
```

Reply:
removed unwanted variables.